@Alyx/newExp.m

```diff
@@ -62,26 +62,22 @@
 
 % Main repository is the reference location for which experiments exist
 [expPath, expRef] = dat.expPath(subject, floor(expDate), expSeq, 'main');
-% ensure a unique `expRef` and `expPath` can be made
-assert(~any(file.exists(expPath)), ...
-  sprintf(['Something went wrong as experiment folders already exist for'...
-  '\n"%s" in \n%s or \n%s. \nIf these folders are empty, delete them and '...
-  'try again.'], expRef, expPath{:}));
-
-% Try making new directories. If this fails, remove empty directories that
```

Comment:
I think something like this comment should be left in as well

Reply:
There are lots of details required in explaining some errors, for instance the solution to mkdirFailed could be a permissions problem or could be a drive access problem, and for the latter it's informative to know that the credentials sometimes need re-entering by accessing the drive through the File Explorer first.  As we discussed before, an FAQ and troubleshooting guide where people can read lengthy troubleshooting info is preferable because you can go into detail on the issue, for instance with a step-by-step guide on resolving the problem.  That's not appropriate in an error message.  In the future some of the information in the troubleshooting guide will be migrated into the code in the form of a Solution object, but for now it is much better to collect this stuff into one place so that we can easily implement this later.  Finally, because this problem usually arises because the folder is created first in the local repo, this won't occur very often in the future because the empty folder is removed in the catch block.

I've already updated the troubleshooting document and might push it to a separate branch when I've added the test for `dat.newExp`.

Reply:
Sure, I agree that lengthy error messages as you describe above shouldn't be in code. In this case, this is simply a comment that describes what the below `try` block is doing: I think this documentation is helpful. Where is latest version of the troubleshooting doc?

Reply:
There are inline comments for this block, it's just they're different from yours so GitHub marks this line as deleted.